src/sentences/dbs.rs

```diff
@@ -0,0 +1,104 @@
+use nom::{character::complete::char, combinator::opt, number::complete::float};
+
+use crate::{Error, NmeaSentence, ParseResult, SentenceType};
+
+/// DBS - Depth Below Surface
+///
+/// <https://gpsd.gitlab.io/gpsd/NMEA.html#_dbs_depth_below_surface>
+/// ```text
+///         1   2 3   4 5   6 7
+///         |   | |   | |   | |
+///  $--DBS,x.x,f,x.x,M,x.x,F*hh<CR><LF>
+/// Field Number:
+///     1. Water depth, feet
+///     2. f = feet
+///     3. Water depth, meters
+///     4. M = meters
+///     5. Water depth, Fathoms
+///     6. F = Fathoms
+///     7. Checksum
+/// In real-world sensors, sometimes not all three conversions are reported.
+/// So you might see something like `$SDDBS,,f,22.5,M,,F*cs`
+///
+/// Examples:
+/// * `$DBS,x.x,f,x.x,M,x.x,F*hh<CR><LF>`
+/// ```
```

Comment:
```suggestion
/// ```
/// In real-world sensors, sometimes not all three conversions are reported.
/// So you might see something like `$SDDBS,,f,22.5,M,,F*cs`
///
/// Examples:
/// * `$DBS,x.x,f,x.x,M,x.x,F*hh<CR><LF>`
```